validation/src/main/scala/hmda/validation/engine/lar/syntactical/LarSyntacticalEngine.scala

```diff
@@ -0,0 +1,28 @@
+package hmda.validation.engine.lar.syntactical
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.api.ValidationApi
+import hmda.validation.engine.lar.LarCommonEngine
+import hmda.validation.rules.lar.syntactical.{ S010, S011, S020, S040 }
+
+trait LarSyntacticalEngine extends LarCommonEngine with ValidationApi {
+
+  def checkSyntactical(lar: LoanApplicationRegister): LarValidation = {
+    val checks = List(
+      S010,
+      S020
+    ).map(check(_, lar))
+
+    validateAll(checks, lar)
+  }
+
+  def checkSyntacticalIterable(lars: Iterable[LoanApplicationRegister]): LarsValidation = {
```

Comment:
Could we rename this method so it doesn't expose the implementation? Maybe we could use `Collection` instead of `Iterable`.